src/pages/Room/CalendarPageLogic.jsx

```diff
@@ -9,7 +9,9 @@ import { roomService } from "services";
 import { Grid, withStyles } from "@material-ui/core";
 import { CalendarGrid } from "./CalendarGrid";
 import { ModalFormConsumer } from "providers";
+import { EasterEgg } from "./components/EasterEgg";
 import classNames from "classnames";
+import { ESPIPE } from "constants";
```

Comment:
what is `ESPIPE`?

Reply:
auto import added it, I don't know what it is, I removed it